conduit/api/base.py

```diff
@@ -872,6 +872,16 @@ def _to_basic_type(self, obj, field):
         if isinstance(field, models.ManyToManyField):
             return eval(field.value_to_string(obj))
 
+        if isinstance(field, models.related.RelatedObject):
+            for virtual_field in self.Meta.model._meta.virtual_fields:
+                if virtual_field.name == 'content_object':
```

Comment:
This obviously needs to not be hardcoded. It seems like exposing this field might require it's own field where we specify the name of the attribute to look up. Otherwise, I'm wondering if we're better off just hiding this attribute from the response data.